pkgs/applications/misc/slic3r/default.nix

```diff
@@ -30,11 +33,16 @@ stdenv.mkDerivation rec {
   };
 
   buildPhase = ''
+    export BOOST_INCLUDEPATH="${boost.dev}/include"
+    export BOOST_LIBRARYPATH="${boost}/lib"
     export SLIC3R_NO_AUTO=true
     export PERL5LIB="./xs/blib/arch/:./xs/blib/lib:$PERL5LIB"
+    export LD=g++
```

Comment:
I would propose to change this line to:
``export LD=$CXX``
or 
``unset LD``
both should work :)